auth/src/main/java/com/firebase/ui/auth/viewmodel/SignInViewModelBase.java

```diff
@@ -19,11 +19,11 @@ protected SignInViewModelBase(Application application) {
     }
 
     @Override
-    protected void setResult(Resource<IdpResponse> output) {
+    public void setResult(Resource<IdpResponse> output) {
         super.setResult(output);
     }
 
-    protected void handleSuccess(@NonNull IdpResponse response, @NonNull AuthResult result) {
+    public void handleSuccess(@NonNull IdpResponse response, @NonNull AuthResult result) {
```

Comment:
Out of curiosity: why are these public now?

Reply:
I'm calling those from AuthMethodPicking activity now :/